test/index/crossrepo/main.snapshot.cc

```diff
@@ -0,0 +1,11 @@
+  #include "external/dep1/dep1.h"
+//^ definition main v0 $`<file>/main.cc`/
+//         ^^^^^^^^^^^^^^^^^^^^^^ reference dep1 v1 $`<file>/dep1.h`/
+  
+  int main(int, char **) {
+//    ^^^^ definition main v0 $main(afcd3168fc1188a4).
+    dep1::f();
+//  ^^^^ reference dep1 v1 $dep1/
+//        ^ reference [..] dep1/f(49f6e7a06ebc5aa8).
```

Comment:
FIXME: This needs to have a `dep1 v1` prefix

Reply:
I think the problem is coming up because we're not emitting the correct external symbol. This is reflected in the `external_symbols.snapshot.cc` too, which is lacking `dep1 v1`